hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -451,6 +456,21 @@ private boolean startsWith(byte[] firstArray, byte[] secondArray) {
   public boolean isVolumeEmpty(String volume) throws IOException {
     String volumePrefix = getVolumeKey(volume + OM_KEY_PREFIX);
 
+    if (bucketTable instanceof TypedTable) {
```

Comment:
When bucketTable is not an instance of TypedTable. As only TypedTables have Cache.
This is more like a safer check, in a case in future if someone changes the bucketTable to RDBTable, we don't need to do the logic inside this if check. Because  cacheIterator throws NotImplementedException for RDBTable.

Reply:
> This is more like a safer check, in a case in future if someone changes the bucketTable to RDBTable

But that will change all of our assumptions about how HA requests work correct? Perhaps it is safer if we fail it rather than ignore it.

Reply:
Okay, will update it